inspire_dojson/model.py

```diff
@@ -47,6 +52,39 @@ def do(self, blob, **kwargs):
 
         return result
 
+    def over(self, name, *source_tags):
+        def decorator(creator):
+            return super(FilterOverdo, self).over(name, *source_tags)(self._wrap_exception(creator, name))
+
+        return decorator
+
+    @staticmethod
+    def _wrap_exception(rule, name):
+        @wraps(rule)
```

Comment:
removing this makes the test fail, because `OverDo` is using the name of the function internally.